data_steward/validation/main.py

```diff
@@ -31,10 +31,13 @@
 from validation import hpo_report
 from tools import retract_data_bq, retract_data_gcs
 from io import open
+from curation_logging.curation_gae_handler import begin_request_logging, end_request_logging
 
 PREFIX = '/data_steward/v1/'
 app = Flask(__name__)
 
+logging.getLogger(__name__).setLevel(logging.INFO)
```

Comment:
@ChaoPang not sure but might there be something like a race condition associated with this statement and `curation_logging.curation_gae_handler.initialize_logging` [here](https://github.com/all-of-us/curation/pull/327/files#diff-2170487847225d1ade922cbb64ff6985R550)?

Reply:
Actually this line of code would not work because it's set `logging.INFO` to the main.py logger but when we log information, we use the root logger instead of logging.getLogger(__name__)

Reply:
@ChaoPang will you remove the line then?

Reply:
This has been removed